package.json

```diff
@@ -7,7 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "clean": "rm -rf build",
     "db:migrate": "sequelize db:migrate",
-    "db:undo:migrate": "sequelize db:migrate:undo:all"
+    "db:undo:migrate": "sequelize db:migrate:undo:all",
+    "dev": "babel-watch server.js"
```

Comment:
Hi, can we move the dev script right before `test`?

Reply:
Of course, I'll get right on it!

Reply:
Also, let's change your commit message from `finished` to `finishes`.
Good job!

Reply:
Commit message is changed.